### ISSUE TITLE
Can you add some screenshots?

### ISSUE THREAD

**Issue by moonvader** [gauravkatoch007/banana#9]
Your sample app not working with Swift 3 out of the box - maybe it is time to update it?
Also It will be very helpful if you can add some screenshots to project description.

**Comment by masterfego** [comment]
I agree with this issue. Very important to show screenshots when you deal with UI
